app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +17,14 @@
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/AppTheme">
+
+        <meta-data
+            android:name="com.google.android.maps.v2.API_KEY"
+            android:value="AIzaSyCXtJj9O1UcfvWMNE0HELWz9HNH-WBOgEU" />
+
+        <meta-data android:name="com.google.android.gms.version"
+            android:value="@integer/google_play_services_version" />
+
```

Comment:
I set us up with a key in google play services. But I just realized we'll need to do:

http://guides.codepath.com/android/Google-Maps-Fragment-Guide#map-access-across-computers

for this to work on all our machines. I'll do that before merging.